XML/TestCases/FunctionalTests-XDP.xml

```diff
@@ -0,0 +1,17 @@
+<TestCases>
+    <test>
+        <testName>VERIFY-XDP-COMPLIANCE</testName>
+        <testScript>VERIFY-XDP-COMPLIANCE.ps1</testScript>
+        <setupType>OneVM2NIC</setupType>
+        <OverrideVMSize>Standard_DS4_v2</OverrideVMSize>
+        <AdditionalHWConfig>
+            <Networking>SRIOV</Networking>
+        </AdditionalHWConfig>
+        <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\XDPDumpSetup.sh</files>
+        <Platform>Azure</Platform>
+        <Category>Functional</Category>
+        <Area>XDP</Area>

```

Comment:
Can you put this test in FunctionalTests-Network.xml?

Reply:
Yes I can, but XDP is new feature and according to [guidelines](https://github.com/LIS/LISAv2/blob/master/Documents/How-to-use.md#xml-files-in-xmltestcases-folder) we should keep it separate.
Also, There are 8 test-cases for this feature, so I think it would look cleaner by keeping them separate just like DPDK test-cases.

Reply:
OK, then let's keep it separate from Network.